charts/agent-control-deployment/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart to install New Relic Agent Control on Kubernetes
 
 type: application
 
-version: 0.0.76
+version: 0.0.77
```

Comment:
it is not related to this PR, but can you change the tag of the system identities from latest to [0.0.10](https://hub.docker.com/layers/newrelic/agent-control-system-identity-registration/0.0.10/images/sha256-b14de0e0da9c08560ca63f0fedd45700d2956a7c1bf357122f04021564480ed5)?